test-cases/gutenberg/columns.md

```diff
@@ -0,0 +1,287 @@
+# Columns Block - Test Cases
+
+--------------------------------------------------------------------------------
+
+##### TC001
+
+### Columns block is added with default number of Column
+
+- Add a `Columns` block
+- Expect `Columns` to have **two** `Column` block inside
+
+Expected look:  
+![DefaultColumnNoStacked](../resources/columns-default-number.png)
+![DefaultColumnNoHorizontal](../resources/columns-default-number-horizontal.png)
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+### Check if Column placeholders is visible for the unselected state
+
+- Add a `Columns` block
+- Deselect
+- Expect to see placeholder for each `Column` added in `Columns`
+
+Expected look:  
+![PlaceholderStacked](../resources/columns-placeholder-root.png)
+![PlaceholderHorizontal](../resources/columns-placeholder-root-horizontal.png)
+
+--------------------------------------------------------------------------------
+
+##### TC003
+
+### Adding Column with AppenderButton
+
+- Add a `Columns` block
+- Having `Columns` block selected press `AppenderButton`
+- Expect new `Column` to be added
+
+Expected look:  
+![Appender](../resources/columns-appender.png)
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Adding Column with Inserter (currently disabled)
```

Comment:
We'll need to update this case now that it is enabled

Reply:
On the second thought we can also remove this test case, because it is inserter's default behavior

Reply:
2nd options seems better for me as well